modules/govuk_postgresql/templates/usr/local/bin/wal-e_env_sync

```diff
@@ -13,20 +13,23 @@ set -e
 # Redirect stdout and stderr to syslog
 exec 1> >(/usr/bin/logger -s -t $(basename $0)) 2>&1
 
+# Disable Puppet so PostgreSQL isn't automatically restarted
+govuk_puppet --disable
```

Comment:
Agreed. If we already have puppet disabled on the host then this job will reenable it no matter the previous reason.